src/renderer.rs

```diff
@@ -15,7 +14,7 @@ pub const DEFAULT_WIREFRAME_COLOR: [f32; 4] = [0.0, 1.0, 1.0, 1.0];
 
 #[repr(C)]
 #[derive(Copy, Clone, PartialEq, Debug)]
-pub struct GpuMatrix4(pub cgmath::Matrix4<f32>);
+pub struct GpuMatrix4(pub Matrix4<f32>);
```

Comment:
I think I'd rather keep the cgmath:: here even though it's not strictly necessary so it's easier to tell which type of matrix is being used since there's a bunch of different math types in the project
```suggestion
pub struct GpuMatrix4(pub cgmath::Matrix4<f32>);
```

Reply:
Ok! Maybe one day we try to standardize everything using only one math crate or using the mint crate.

Reply:
👍